examples/request_configuration.py

```diff
@@ -15,14 +15,23 @@
 
 """Example of request config"""
 from tensorbay import GAS
-from tensorbay.client.requests import config
-from tensorbay.opendataset import LISATrafficLight
+from tensorbay.client import config
+from tensorbay.dataset import Data, Dataset
 
+# Set "config" as needed before authorize a client object.
 config.timeout = 40
 config.max_retries = 4
 
-gas = GAS(access_key="Accesskey-***")
+gas = GAS("<YOUR_ACCESSKEY>")
 
-dataset = LISATrafficLight("/path/to/dataset")
-gas.upload_dataset(dataset)
+# Organize the local dataset by the "Dataset" class before uploading.
+dataset = Dataset("DatasetName")
+
+segment = dataset.create_segment()
+segment.append(Data("0000001.jpg"))
+segment.append(Data("0000002.jpg"))
+
+# Configured dataset client is ready to upload dataset.
```

Comment:
`Configured dataset` is wrong, the configuration is not for the dataset. The configuration is for the client to adjust the sending requests behavior.